pull-scan.sh

```diff
@@ -20,7 +20,7 @@ else
 fi
 
 # Scans the image
-tmas scan "$(if [ "$MALWARE_SCAN" = true ]; then echo "--malwareScan"; fi)" -r "$REGION" docker-archive:"$IMAGE_TARBALL" "$(if [ "$SBOM" = true ]; then echo "--saveSBOM"; fi)" > "$SCAN_RESULT_ARTIFACT"
+tmas scan "$(if [ "$VULNERABILITY_SCAN" = true ]; then echo "-V"; fi)" "$(if [ "$MALWARE_SCAN" = true ]; then echo "-M"; fi)" "$(if [ "$SECRETS_SCAN" = true ]; then echo "-S"; fi)" -r "$REGION" docker-archive:"$IMAGE_TARBALL" "$(if [ "$SBOM" = true ]; then echo "--saveSBOM"; fi)" > "$SCAN_RESULT_ARTIFACT"
 
```

Comment:
Hello, as an FYI, lines 36 onwards will also need modification, as the report format has changed starting in TMAS v2+, and vulnerability results live under vulnerabilities.X instead of vulnerability.X

Checkout the WHATS_NEW.md that is shipped with the TMAS binary for more details on the report format changes. 

Thanks in advance!